### ISSUE TITLE
OIDC q.auth details

### ISSUE THREAD

**Issue by AlexWennerDT** [h2oai/wave#745]
The only details of the q.auth I am familiar with currently are q.auth.subject and q.auth.token. Wondering if there were more details you could retrieve from the oidc provider?

**Comment by lo5** [comment]
Yes, we can improve this feature to pass on additional attributes from the provider. Is there anything specific you are looking for? We could also make this configurable.

**Comment by AlexWennerDT** [comment]
I would assume configurable would be the most dynamic being that it works with external providers! That would be an awesome feature

**Comment by lo5** [comment]
Blocked by  #481
